Dockerfile

```diff
@@ -245,9 +245,21 @@ USER buildbot
 RUN /bin/bash -c 'phpbrew init && source ~/.phpbrew/bashrc && phpbrew install 5.6 +default && \
     phpbrew app get composer'
 
+################################################################################
+#
+# Emacs Cask
+#
+################################################################################
 USER root
+RUN add-apt-repository ppa:kelleyk/emacs
```

Comment:
>  If the 24->25 upgrade is really affecting build

sorry I'm not following, this is not an upgrade, the Ubuntu 14 docker don't have any emacs preinstalled, how it's gonna affect v24 build?